db/indices.c

```diff
@@ -423,6 +423,24 @@ int add_record_indices(struct ireq *iq, void *trans, blob_buffer_t *blobs,
                 vgenid = 0; // no need to verify again
             }
 
+            if (iq->vfy_idx_track) {
+                char * store_key;
+                store_key = pool_getablk(iq->vfy_idx_pool);
+                memcpy(store_key, &ixnum, sizeof(int));
+                memcpy(store_key + sizeof(int), key, ixkeylen);
```

Comment:
The table itself needs to be part of the key, I believe (can probably use iq->usedb->dbs_idx), otherwise we may report false positives from upserts that write the same key value to the same ixnum but on different tables.

Reply:
Yes that's true. Will update, thanks :)